frontend/src/i18n/translation.json

```diff
@@ -5743,6 +5743,22 @@
         "de": "API-Schlüssel ermöglichen es Ihnen, sich programmatisch bei der OpenHands-API zu authentifizieren. Halten Sie Ihre API-Schlüssel sicher; jeder mit Ihrem API-Schlüssel kann auf Ihr Konto zugreifen. Weitere Informationen zur Verwendung der API finden Sie in unserer <a>API-Dokumentation</a>.",
         "uk": "Ключі API дозволяють вам програмно автентифікуватися за допомогою API OpenHands. Зберігайте свої ключі API в безпеці; будь-хто, хто має ваш ключ API, може отримати доступ до вашого облікового запису. Для отримання додаткової інформації про використання API, перегляньте нашу <a>документацію API</a>."
     },
+    "SETTINGS$OPENHANDS_API_KEY_HELP": {
+        "en": "You can find your OpenHands API Key in the `API Keys` tab of OpenHands Cloud.",
+        "ja": "OpenHands APIキーはOpenHands Cloudの`APIキー`タブで確認できます。",
+        "zh-CN": "您可以在OpenHands Cloud的`API密钥`标签页中找到您的OpenHands API密钥。",
+        "zh-TW": "您可以在OpenHands Cloud的`API密鑰`標籤頁中找到您的OpenHands API密鑰。",
+        "ko-KR": "OpenHands API 키는 OpenHands Cloud의 `API 키` 탭에서 찾을 수 있습니다.",
+        "no": "Du kan finne din OpenHands API-nøkkel i `API-nøkler`-fanen i OpenHands Cloud.",
+        "it": "Puoi trovare la tua chiave API OpenHands nella scheda `Chiavi API` di OpenHands Cloud.",
+        "pt": "Você pode encontrar sua chave de API OpenHands na guia `Chaves de API` do OpenHands Cloud.",
+        "es": "Puede encontrar su clave API de OpenHands en la pestaña `Claves API` de OpenHands Cloud.",
+        "ar": "يمكنك العثور على مفتاح API الخاص بـ OpenHands في علامة التبويب `مفاتيح API` في OpenHands Cloud.",
+        "fr": "Vous pouvez trouver votre clé API OpenHands dans l'onglet `Clés API` d'OpenHands Cloud.",
+        "tr": "OpenHands API Anahtarınızı OpenHands Cloud'un `API Anahtarları` sekmesinde bulabilirsiniz.",
+        "de": "Sie finden Ihren OpenHands API-Schlüssel im Tab `API-Schlüssel` von OpenHands Cloud.",
+        "uk": "Ви можете знайти свій ключ API OpenHands у вкладці `Ключі API` OpenHands Cloud."
+    },
```

Comment:
@OpenHands We should add link "API Keys Tab" to this URL: https://app.all-hands.dev/settings/api-keys